lib/xapi-stdext-pervasives/pervasiveext.mli

```diff
@@ -52,3 +52,4 @@ val ignore_bool : bool -> unit
 val ( ++ ) : ('b -> 'c) -> ('a -> 'b) -> 'a -> 'c
```

Comment:
We could also deprecate `(++) f g x`, to be replaced by `f @@ g @@ x`

Reply:
Sometimes it cannot be replaced because the function is partially applied. It can be defined locally, however.